### DISCUSSION TITLE
could you release on flathub please?

### DISCUSSION THREAD

**Discussion by ngmisl** [massCodeIO/massCode#151] [Ideas]
see topic, thank you!

also, do you accept monero / other crypto as donations?

**Comment by antonreshetov** [comment]
@ngmisl there is an address on BTC - `bc1qxrgea7epgt2qu72ylp6lg2n7dm59zcqvxcvaja`

**Comment by vmgustavo** [comment]
will you release on flathub?

**Comment by antonreshetov** [comment]
@vmgustavo see #87
